calico-cloud_versioned_docs/version-3.16/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,111 +2,97 @@
+**Why do I have images with `unknown` results**?
 
-![scan-settings](/img/calico-cloud/scan-settings.png)
+If scan results contain `unknowns` it can mean:
+- The Image Assurance scanner is not enabled and running
+- The scanner has not yet reported results yet
 
-To change the CVSS threshold values, note the following:
+**How long does it take for scan results?**
 
-- Changes to threshold values take effect immediately and alter the scan results for images already in the system
-- If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously being blocked, to now be deployed or vice versa.
+Anywhere from 2-10 minutes. The variability occurs because the scanner watches, reports, and then sleeps to balance efficiency and resources.
 
-**Enable Runtime View**
+**What is Download Scanner?**
 
-You can configure your scan settings to directly monitor pods that are running in your Kubernetes clusters.
-When enabled, Runtime View checks image information from the active container images in all of your connected clusters.
-These images are correlated with the images you have recorded in the Image Assurance system.
+This link goes to steps to integrate the Image Assurance scanner as part of your build pipeline. If you want to scan images only in your Kubernetes cluster, see [Enable automatic scanning inside a cluster](./scanners/cluster-scanner).
 
-To enable Runtime View, select the **Enable Runtime View** checkbox in the **Scan Results** tab.
-You can clear the checkbox to disable Runtime View.
-After disabling Runtime View, some runtime information for existing images will remain in the system.
+### Scan settings panel
 
-:::note
+**What is the Enable Runtime View checkbox?**
 
-If your cluster does not use the default containerd socket path (`/run/containerd/containerd.sock`), then you must change the path to allow the Runtime View service to collect image information.
+Runtime view allows you to view vulnerable images that are running in your cluster. You can then remediate the vulnerabilities using the CVSS scores as a guideline to triage CVEs.
 
-To update the CRI socket path for a cluster, run the following command:
+If you disable Runtime View, some runtime information for existing images will remain in the system.
 
-```bash
-kubectl patch imageassurance default --type='merge' -p '{"spec":{"criSocketPath":"<path-to-cri-socket>"}}'
-```
-See [the installation reference](../reference/installation/api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec) for more details on the ImageAssurance CR.
+**What are the `API URL` and `API Token` fields used for?**
 
-:::
+They are used to authenticate the connection to the Image Assurance scanner if you have integrated Image Assurance scanner in your build pipeline.
```

Comment:
I don't think "connection" is the right word because it implies a long running communication channel with the CLI scanner. 

I would just say "They are used to authenticate **API requests** between the CLI scanner to the Image Assurance API ...".

Reply:
Fixed as noted.